src/main/java/frc/robot/subsystems/drivetrain/Tank.java

```diff
@@ -0,0 +1,98 @@
+package frc.robot.subsystems.drivetrain;
+
+import static frc.robot.Constants.TankConstants.LEFT_MAIN;
+import static frc.robot.Constants.TankConstants.LEFT_SECONDARY;
+import static frc.robot.Constants.TankConstants.RIGHT_MAIN;
+import static frc.robot.Constants.TankConstants.RIGHT_SECONDARY;
+
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX; // internal notfalcon CIM motors
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.motorcontrol.MotorUtil;
+
+public class Tank extends Drivetrain {
+
+    private final WPI_TalonSRX left = MotorUtil.createTalonSRX(LEFT_MAIN); // left motor
+    private final WPI_TalonSRX left2 = MotorUtil.createTalonSRX(LEFT_SECONDARY); // left motor
+
+    private final WPI_TalonSRX right = MotorUtil.createTalonSRX(RIGHT_MAIN); // right motor
+    private final WPI_TalonSRX right2 = MotorUtil.createTalonSRX(RIGHT_SECONDARY); // right motor
+  
+    double leftDrive;
+    double rightDrive;
+    double xPower;
+    double yPower;
+    double angularPower;
+
+  public Tank() {
+    super();
+
+    left2.follow(left);
+    right2.follow(right);
+
+    left.setNeutralMode(NeutralMode.Brake);
+    right.setNeutralMode(NeutralMode.Brake);
+    right.setInverted(true);
+    right2.setInverted(true);
+
+    
+
+
+  }
+
+  public void updateDrivePowers(double forwardComponent, double sideComponent, double angularPower){
+    // POSSIBLE/CONCEPT -- double theta = Math.atan(yPower/xPower); // if the robot needs to move some amount laterally, it can rotate to this angle and move in that direction
+    
+    leftDrive = xPower + yPower;
+    rightDrive = xPower - yPower;
```

Comment:
if these are only used in the scope of this method, it makes sense to me to declare them here (rather than as field variables)